internal/repository/tenant_alerting.go

```diff
@@ -40,12 +44,17 @@ type TenantAlertingAPIRepository interface {
 	DeleteTenantAlertGroup(tenantId string, id string) error
 }
 
+type TenantAlertEmailGroupForSend struct {
+	TenantId pgtype.UUID `json:"tenantId"`
+	Emails   string      `json:"emails"`
```

Comment:
```suggestion
	Emails []string `validate:"required,dive,email,max=255"`
```

(assuming this is meant to be a list of emails) - better to validate with the string array and concat in the repository.